locale/en/blog/vulnerability/jan-2022-security-releases.md

```diff
@@ -1,12 +1,82 @@
 ---
-date: 2022-01-04T16:00:00.000Z
+date: 2022-01-10T16:00:00.000Z
```

Comment:
We may need to move this forward if we want this to appear above the release posts?

Reply:
+1, We should move it to be right now

Reply:
2022-01-11T00:48:00:00.000Z would probably make sense.